jwt.js

```diff
@@ -96,10 +97,32 @@ function fastifyJwt (fastify, options, next) {
     cookie: cookie,
     sign: sign,
     verify: verify
-  })
-  fastify.decorateRequest('user', null)
-  fastify.decorateRequest('jwtVerify', requestVerify)
-  fastify.decorateReply('jwtSign', replySign)
+  }
+
+  let jwtVerifyName = 'jwtVerify'
+  let jwtSignName = 'jwtSign'
+  if (namespace) {
+    if (!fastify.jwt) {
+      fastify.decorate('jwt', Object.create(null))
+      fastify.decorateRequest('user', null)
+      fastify.decorateRequest('jwt', Object.create(null))
```

Comment:
Where does it use? same as `.decorateReply`

Reply:
I don't understand the question.

Reply:
@mcollina `jwt` is decorate on `request` and `reply` object but it never used.

Reply:
Good spot, it was a misprint